### ISSUE TITLE
[Request] Add ability to set widget background color and opacity.

### ISSUE THREAD

**Issue by Xasweil** [termux/termux-widget#10]
This is an optional function, but for visual satisfaction it would be nice to configure at least the transparency of the widget background. Accordingly when changing the background you should also add the ability to change the color of text (white for dark background and black for light).

**Comment by dellasorte** [comment]
yep. it would be great to change widget colors

**Comment by nvllsvm** [comment]
Dupe of https://github.com/termux/termux-widget/issues/20

**Comment by ioogithub** [comment]
I really need this as well. This widget gets a coveted place on my home screen however I have a 100% dark theme phone. This is the only piece of critical software that I cannot change the bright white background to a dark shade.

Please include a preset dark mode version of the widget or else allow the user to change the colors.

**Comment by lfg6000** [comment]
this just an up vote for the enhancement request
- using a dark theme on android and termux itself has a black background and the text is white and this is beautiful. 
- the termux:Wiidget with a list of scripts has a white background and black text and it seems so very bright. 
- maybe at some point the widget with a list of scripts could have a background with white text option.

**Comment by agnostic-apollo** [comment]
The widget app was rewritten locally a while back and should be possible to add color configuration. Will be couple of months though before its ready as depends on termux-app changes which aren't ready yet.

**Comment by xfechx** [comment]
Is there a workaround for this, while the feature is implemented to change background color, opacity and text color to termux:widget?

**Comment by agnostic-apollo** [comment]
No, you could potentially modify the source code and rebuild the app.
